packages/cli/create-strapi-starter/src/utils/fetch-npm-starter.ts

```diff
@@ -0,0 +1,87 @@
+import path from 'path';
+import execa from 'execa';
+import chalk from 'chalk';
+import stopProcess from './stop-process';
+import type { Options, PackageInfo } from '../types';
+
+/**
+ * Gets the package version on npm. Will fail if the package does not exist
+ */
+async function getPackageInfo(packageName: string, options?: Options): Promise<PackageInfo> {
+  const { useYarn } = options ?? {};
+
+  // Use yarn if possible because it's faster
+  if (useYarn) {
+    const { stdout } = await execa('yarn', ['info', packageName, '--json']);
+    const yarnInfo = JSON.parse(stdout);
+    return {
+      name: yarnInfo.data.name,
+      version: yarnInfo.data.version,
```

Comment:
If the package doesn't exist, yarn returns `{"type":"error","data":"Received invalid response from npm."}` so this will throw a 'property doesn't exist on data' instead of a "No match..." like the npm version below

Reply:
Happy to fix it but that's probably already an existing bug :)

Reply:
Haha, yeah, I realized after I hit submit.